### PR TITLE
MUSARK-998 - feature/analysis missing fields

### PR DESCRIPTION
**Pull request by ingarabr**
Related to MUSARK-998

Analysis collection:
- case number
- reason
- status

Sample created:
- external links

### CODE REVIEW

File: service_management/app/models/analysis/SampleStatuses.scala [hunk 1]
```diff
@@ -16,6 +16,8 @@ object SampleStatuses {
         case Destroyed.key    => Some(Destroyed)
         case Contaminated.key => Some(Contaminated)
         case Prepared.key     => Some(Prepared)
+        case Discarded.key    => Some(Discarded)
+        case Dehigdrated.key  => Some(Dehigdrated)
```
Comment (kpmeen) [inline service_management/app/models/analysis/SampleStatuses.scala]: I think you mean `Dehydrated`, or?

File: service_management/app/models/analysis/SampleStatuses.scala [hunk 2]
```diff
@@ -33,14 +35,18 @@ object SampleStatuses {
...
         case JsSuccess(bad, p)              => JsError(p, s"Unknown sample status code $bad")
         case err: JsError                   => err
       }
     }
 
     implicit val writes: Writes[SampleStatus] = Writes(ss => JsNumber(ss.key))
   }
-
+  /*
+: Ok, Oppbrukt, Kontaminert, Preparert, Kassert og Uttørket
```
Comment (kpmeen) [inline service_management/app/models/analysis/SampleStatuses.scala]: I would add this as part of the comment for the `SampleStatuses` object, I think.

File: service_management/conf/evolutions/default/1.sql [hunk 3]
```diff
@@ -60,6 +60,8 @@ CREATE TABLE MUSARK_ANALYSIS.EVENT (
   part_of         NUMBER(20),
   object_uuid     VARCHAR2(36),
   note            VARCHAR2(500),
+  status          NUMBER,
```
Comment (kpmeen) [inline service_management/conf/evolutions/default/1.sql]: This should be `INTEGER` to ensure we're only adding integer numbers.

File: docker/dev/oracle-init.sql [hunk 4]
```diff
@@ -421,6 +421,8 @@ CREATE TABLE MUSARK_ANALYSIS.EVENT (
   part_of         NUMBER(20),
   object_uuid     VARCHAR2(36),
   note            VARCHAR2(500),
+  status          NUMBER,
```
Comment (kpmeen) [inline docker/dev/oracle-init.sql]: This should be `INTEGER`.

File: service_management/test/resources/evolutions/test/1.sql [hunk 5]
```diff
@@ -61,6 +61,8 @@ CREATE TABLE MUSARK_ANALYSIS.EVENT (
   part_of         NUMBER(20),
   object_uuid     VARCHAR2(36),
   note            VARCHAR2(500),
+  status          NUMBER,
```
Comment (kpmeen) [inline service_management/test/resources/evolutions/test/1.sql]: This should be `INTEGER`.

**Review by kpmeen — CHANGES_REQUESTED** [review]

File: service_management/app/models/analysis/SampleStatuses.scala (hunk 2 above)
Comment (ingarabr) [reply service_management/app/models/analysis/SampleStatuses.scala]: Note to self, remove temp comments before committing!

**Review by kpmeen — APPROVED** [review]

**Pull request merged**